awx/main/managers.py

```diff
@@ -101,11 +101,13 @@ class InstanceManager(models.Manager):
     instance or role.
     """
 
+    UUID_DEFAULT = '00000000-0000-0000-0000-000000000000'
```

Comment:
you should combine this with

https://github.com/ansible/awx/blob/6a17e5b65bf9421c2e650d23817c79ce81248c6f/awx/main/models/ha.py#L62

Reply:
you're right, good call.

Reply:
is that along the lines of what you were thinking @AlanCoding ?

Reply:
oh yeah, looks great, sorry for the delayed reply.